scripts/data/pyLoadObservations/pyLoadObservations.sh

```diff
@@ -1,2 +1,3 @@
 #!bin/sh
-docker-compose -f /scripts/pyLoadObservations/docker/docker-compose.yaml run pc-dock python -u /home/jovyan/scripts/pyLoadObservations/getocc/get_occ.py ${1}
\ No newline at end of file
+docker-compose -f /scripts/data/pyLoadObservations/docker/docker-compose.yaml build
+docker-compose -f /scripts/data/pyLoadObservations/docker/docker-compose.yaml run pc-dock python -u /home/jovyan/scripts/data/pyLoadObservations/getocc/get_occ.py ${1}
```

Comment:
Isn't the build implicit if you do run ?

Reply:
My tests said no : if I modify the dockerfile and use run, it is not rebuilt. 
I know using run pulls precompiled images, so maybe it builds but only when never built before?